README.md

```diff
@@ -62,80 +75,123 @@ cloud.name=${CLOUD_NAME}
+by this URL: http://localhost:8080/api/0/apidocs/index.html
+
+Detail info about local running or about other running options you can read in 
+[hom-dev](https://github.com/ita-social-projects/Home/tree/dev/home-dev) package.
 
 ---
 
-## Documentation
-- You can find OpenApi specification for the project [here](https://home-project-academy.herokuapp.com/api/0/apidocs/index.html)
+### Docker images
+Our [images](https://hub.docker.com/u/homeacademy) on Docker Hub:
+
+- ***data-migration*** - this image starts a data-migration for database in docker container
+
+- ***home-application*** - this image starts an application in docker container
 
 ---
 
-## Contributing
+## Documentation
 
-Our images on Docker Hub
+- You can find OpenApi specification for the
+  project [here](https://home-project-academy.herokuapp.com/api/0/apidocs/index.html).
 
-    https://hub.docker.com/u/homeacademy
+---
 
+## Contributing
 ### Git flow
 > To get started...
 #### Step 1
 
 - **Option 1**
-    - 🍴 Fork this repo!
+  - 🍴 Fork this repo!
 
 - **Option 2**
-    - 👯 Clone this repo to your local machine using `https://github.com/ita-social-projects/Home.git`
+  - 👯 Clone this repo to your local machine using command:
+
+`git clone https://github.com/ita-social-projects/Home.git`
 
 #### Step 2
 
-- **HACK AWAY!** 🔨🔨🔨
+Create your Feature Branch
+
+`git checkout -b name_for_new_branch`
 
 #### Step 3
+Make changes and test
 
-- 🔃 Create a new pull request using <a href="https://github.com/ita-social-projects/Home/compare/" target="_blank">github.com/ita-social-projects/Home</a>.
+#### Step 4
+Open a Pull Request with description of changes
 
 ---
 
 ## Team
-- Technical Expert of the project - @kh0ma
-- Mentor of SoftServe ITA - @DzigMS
+
+- Technical Expert of the project - `@kh0ma`
+- Mentor of SoftServe ITA - `@DzigMS`
 
 Developers that have worked on a project during:
-#### September - December 2020
-- @mykytam
-- @reeztl
-- @Prosperro-de
-- @alexorlenko
-- @omelchenkotimur
-- @Arthurpanda
-
-#### December 2020 - March 2021
-
-- @alexorlenko
-- @Dandria1802
-- @Romanov-Niko
-- @PoLytvy
-- @OlTym
-- @Abhai2016
+
+- `@mykytam`
+- `@reeztl`
+- `@Prosperro-de`
+- `@alexorlenko`
+- `@omelchenkotimur`
+- `@Arthurpanda`
+- `@Dandria1802`
+- `@Romanov-Niko`
+- `@PoLytvy`
+- `@OlTym`
+- `@Abhai2016`
+- `@tex1988`
+- `@denis-litvinov`
+- `@VadymSokorenko`
+- `@annaprok`
+- `@dd994`
+- `@likeRewca`
+- `@Ilya-Ross`
+- `@mitianin`
+- `@BenAvleck`
+- `@Vladyslav-Frolov`
+- `@AriannaLiss`
+- `@kizimov`
+- `@IhorSamoshost`
+- `@Eternal107`
+
+We are also fortunate to have an
+amazing [community of developers](https://github.com/ita-social-projects/Home/graphs/contributors) who help us greatly.
 
 ## Support
 
-Reach out to me at one of the following places!
 
-- Website at <a href="http://Website.com" target="_blank">`Website.com`</a>
-- Facebook at <a href="https://www.facebook.com/LiubomyrHalamaha/" target="_blank">`Liubomyr Halamaha`</a>
 ---
+## Contact
```

Comment:
I think, we should add some contacts and support or just delete this paragraphs.